janusgraph-backend-testutils/src/main/java/org/janusgraph/testutil/JanusGraphAssert.java

```diff
@@ -81,4 +91,57 @@ private static boolean isEmpty(Object obj) {
         throw new IllegalArgumentException("Cannot determine size of: " + obj);
     }
 
+    public static void assertNumStep(int expectedResults, int expectedSteps, GraphTraversal traversal, Class<? extends Step>... expectedStepTypes) {
```

Comment:
nitpick: Can you add Javadoc for this method? The method name itself is not clear enough to understand its usage.